src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -145,12 +145,15 @@ const ArrayItem: React.VFC<
       disabled={isDisabled}
       leftGlyph={<Icon glyph="Trash" />}
       data-cy="delete-item-button"
+      size="small"
     />
   );
   return useExpandableCard ? (
     <StyledExpandableCard
       defaultOpen={!isDisabled}
       data-cy="expandable-card"
+      // Override LeafyGreen's string typing for title so we can include buttons.
+      /* @ts-expect-error */
```

Comment:
Maybe link this here [LG-2193](https://jira.mongodb.org/browse/LG-2193) 🐰

Reply:
Good find 🕵️‍♀️